nemo/collections/multimodal/data/energon/sampleencoder.py

```diff
@@ -0,0 +1,528 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
is it better to call them `sample_encoder.py` and `task_encoder.py`

Reply:
done